Terminal.Gui/Views/Scroll/ScrollButton.cs

```diff
@@ -0,0 +1,142 @@
+﻿#nullable enable
+
+namespace Terminal.Gui;
+
+internal enum VariationMode
```

Comment:
One type per file please. 

Please add API docs even though this is internal to explain intent.

Reply:
Consider just using `NavigationDirection` for this, renaming it to `Direction` to make it generic. 

This way we reduce the number of new types?

Or, `AddOrSubtract` might work too?

Reply:
I opted to use the `NavigationDirection.Backward` and `NavigationDirection.Forward`. Thanks for the suggestion.